### ISSUE TITLE
[Feature] Schedule RayJob

### ISSUE THREAD

**Issue by ammarar** [ray-project/kuberay#1662]
### Search before asking

- [X] I had searched in the [issues](https://github.com/ray-project/kuberay/issues) and found no similar feature requirement.

### Description

Is to add an extra functionality to RayJob to make it able to be running on a schedule (like everyday, every week, etc). This is similar to what Spark operator has, called [ScheduledSparkApplication](https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/blob/master/docs/api-docs.md#sparkoperator.k8s.io/v1beta2.ScheduledSparkApplication)

### Use case

Running a training job that trains a model every 2 weeks. Right now we have a to use Argo to do this which complicates the toolset.

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

**Comment by kevin85421** [comment]
The discussions in https://github.com/ray-project/kuberay/issues/1206 might be relevant.

**Comment by shuangkun** [comment]
Are you using Argo Workflow to create resource templates to arrange Ray Jobs?

**Comment by ammarar** [comment]
@shuangkun No, we use Argo workflow to schedule running jobs on a standing Ray cluster
